scripts/dev_docs.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+set -euo pipefail
+
+KIBANA_DIR=$(cd "$(dirname "$0")"/.. && pwd)
+WORKSPACE=$(cd "$KIBANA_DIR/.." && pwd)/kibana-docs
+export NVM_DIR="$WORKSPACE/.nvm"
+
+DOCS_DIR="$WORKSPACE/docs.elastic.dev"
+
+# These are the other repos with docs currently required to build the docs in this repo and not get errors
+# For example, kibana docs link to docs in these repos, and if they aren't built, you'll get errors
+DEV_DIR="$WORKSPACE/dev"
+TEAM_DIR="$WORKSPACE/kibana-team"
+
+cd "$KIBANA_DIR"
+origin=$(git remote get-url origin || true)
+GIT_PREFIX="git@github.com:"
+if [[ "$origin" == "https"* ]]; then
+  GIT_PREFIX="https://github.com/"
+fi
+
+mkdir -p "$WORKSPACE"
+cd "$WORKSPACE"
+
+if [[ ! -d "$NVM_DIR" ]]; then
+  echo "Installing a separate copy of nvm"
+  git clone https://github.com/nvm-sh/nvm.git "$NVM_DIR"
+  cd "$NVM_DIR"
+  git checkout "$(git describe --abbrev=0 --tags --match "v[0-9]*" "$(git rev-list --tags --max-count=1)")"
+fi
+source "$NVM_DIR/nvm.sh"
+
```

Comment:
If nvm is not installed, we'll run the clones below in $NVM_DIR instead of $WORKSPACE

Reply:
Good catch, thank you